_content/setup-wallets.md

```diff
@@ -0,0 +1,87 @@
+
+If you want to send actual value in your app with an Interledger account, refer to the [Rust implementation of Interledger](http://interledger.rs), or the Java implementation of Interledger at [Hyperledger Quilt](https://www.hyperledger.org/projects/quilt). Because these efforts are open source, your contributions are welcome.
+
+You can also perform these same tasks programmatically. See [Set up Interledger accounts programmaticallly](setup-wallets-programmatically.html).
+
+### Create an Interledger account and get your own payment pointer
+
+This section refers to the Xpring ILP testnet.
+
+1. Open the login page for the Xpring ILP testnet wallet: [https://wallet.ilpv4.dev/](https://wallet.ilpv4.dev/)<br>
+   Read the Terms of Use and the Privacy Policy, as linked on that page.
+2. If you agree with the specified terms and privacy policy, click **Sign in**.<br>
+   You are prompted to sign in with one of Facebook, Github, or Google credentials.
+3. For your login, select the option you prefer.<br>
+   You can create multiple accounts that you control by signing in subsequently with other options.
+4. You might be prompted to log in with your credentials, or if you are already logged in, you might be prompted for further authentication.<br>
+   When you are logged in, you see a page with your payment pointer, in the following format:<br>
+   `$money.ilpv4.dev/<your-account-name>`<br>
+   As indicated by the payment pointer, this wallet uses the Interledger Protocol, v4. Because a new account does not have any test funds, you will want to add value to it.
+5. Click **Make it rain**.<br>
+  Your account now has 10 test XRP. To add more test XRP, continue to click the button until you have as many test XRP as you want.
+6. If you want someone to pay you, provide them with your payment pointer and ask them to use their own payment pointer to send funds to your payment pointer.
+7. If you want to pay someone, request that they provide you with their payment pointer and use their payment pointer to send them funds.<br>
+   You can readily set up multiple accounts to get different payment pointers, and therefore experiment with sending payments in different scenarios where you have ready access to the accounts on both sides.
+
+When you subsequently log in to this testnet again with the credentials that you used previously, you will see the current state of your wallet with the current amount of test XRP it has. Your account can receive test XRP when you are not logged in.   
+
+### Send money between Interledger accounts
+
+When you have funds in your account, you can then send funds to another account.
+
+1. Sign in to the testnet at [https://wallet.ilpv4.dev/](https://wallet.ilpv4.dev/). Look at the **Send** portion of the page on the right.
+2. Enter the number of test XRP, in integer form, that you want to send.
```

Comment:
We may want to make it clear that the amount of XRP in this field not only has to be an integer (I guess we don't support fractional amounts yet), but that it is denominated in whole XRP, **not** in the account asset scale.  Otherwise, this may cause some confusion, especially if a developer has just read about asset scales

Reply:
As discussed with Noah, the GUI-based tutorial does not refer to asset scales at all, nor do I see any way to change it in the GUI. So I'll leave that for now, because I can't see a way to reference asset scale without bringing in programmatic access.

Reply:
Yeah, we definitely don't want to explain the concept of asset scales! Instead of "integer form," maybe this?
```suggestion
2. Enter the number of test XRP, in whole number increments, that you want to send.
```

Reply:
Changed